Packs/Flashpoint/Integrations/Ignite/README.md

```diff
@@ -0,0 +1,1600 @@
+3. Click **Add instance** to create and configure a new integration instance.
+4. To fetch Ignite Alerts refer to the section ["Configuration for fetching Ignite Alerts as an XSOAR Incident".](#configuration-for-fetching-ignite-alerts-as-an-xsoar-incident)
+5. To fetch Ignite Alerts refer to the section ["Configuration for fetching Ignite Compromised Credentials as an XSOAR Incident".](#configuration-for-fetching-ignite-compromised-credentials-as-an-xsoar-incident)
+
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Fetch incidents |  | False |
+    | Incident type |  | False |
+    | Server URL | Server URL to connect to Ignite. | True |
+    | API Key | API key used for secure communication with the Ignite platform. | True |
+    | Maximum number of incidents per fetch | The maximum limit is 200 for alerts and compromised credentials. | False |
+    | First fetch time interval | Date or relative timestamp to start fetching the incidents from. \(Formats accepted: 2 minutes, 2 hours, 2 days, 2 weeks, 2 months, 2 years, yyyy-mm-dd, yyyy-mm-ddTHH:MM:SSZ, etc\). | False |
+    | Fetch Type | Whether to fetch the Ignite alerts or the compromised credentials. Would choose "Compromised Credentials" if nothing selected. | False |
+    | Severity for Incidents | Set the default severity for the incidents using this instance. | False |
+    | Alert Status | Filters the incoming alerts with the provided alert status. | False |
+    | Alert Origin | Filters the incoming alerts with the origin of alert. | False |
+    | Alert Sources | Filters the incoming alerts with the source of alert. | False |
+    | Fetch fresh compromised credentials alerts | Adds the 'is_fresh' flag to compromised credential queries so it only ingests username/password combinations if they haven't been seen before. | False |
+    | Source Reliability | Reliability of the source providing the intelligence data. | False |
+    | Create relationships | Create relationships between indicators as part of Enrichment. | False |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+
+6. Click **Test** to validate the URLs, token, and connection.
+
+## Configuration for fetching Ignite Alerts as an XSOAR Incident
+
+1. Select Fetches incidents.
+2. Under Classifier, select "N/A".
+3. Under Incident type, select "Ignite Alert".
+4. Under Mapper (incoming), select "Ignite Alert - Incoming Mapper" for default mapping.
+5. Enter the connection parameters. (Server URL, API key)
+6. Under the Fetch Type, select "Alerts".
+7. Select "Alert Status" based on your requirement to filter the alerts.By default, it will fetch all status of alerts.
+8. Select "Alert Origin" based on your requirement to filter the alerts.By default, it will fetch all origins of alerts.
+9. Select "Alert Sources" based on your requirement to filter the alerts.By default, it will fetch all sources of alerts.
```

Comment:
```suggestion
7. Select "Alert Status" based on your requirement to filter the alerts. By default, it will fetch all status of alerts.
8. Select "Alert Origin" based on your requirement to filter the alerts. By default, it will fetch all origins of alerts.
9. Select "Alert Sources" based on your requirement to filter the alerts. By default, it will fetch all sources of alerts.
```